src/utils/PlayerState.js

```diff
@@ -169,6 +170,20 @@ class PlayerState {
   static GetNodesVisited(keyPrefix, defaultValue = null) {
     return persistantStorage.get(keyPrefix, KeyConstants.VISIT_ONCE_NODE_LIST, defaultValue);
   }
+
+  static SetSessionReport(keyPrefix, obj) {

```

Comment:
Why is the session report persisted to storage?  It really only has a life during that one view...

Reply:
Just thought it would be consistent, the way most of the other data is cached. I'll get rid of that.